stacks/victoria-metrics-cluster/uninstall.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+STACK="vmoperator"
+NAMESPACE="default"
+
+helm uninstall "$STACK" \
+  --namespace "$NAMESPACE"
+
+# Uninstall VictoriaMetrics Cluster
+if [ -z "${MP_KUBERNETES}" ]; then
+  # with local version of values.yml
+  ROOT_DIR=$(git rev-parse --show-toplevel)
+  VMCLUSTER="$ROOT_DIR/stacks/victoria-metrics-cluster/yaml/vmcluster.yaml"
+  VMAGENT="$ROOT_DIR/stacks/victoria-metrics-cluster/yaml/vmagent.yaml"
+  kubectl delete -f "$VMCLUSTER"
+  kubectl delete -f "$VMAGENT"
+else
+  # with github hosted master version of values.yml
+  kubectl delete -n default -f https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/victoria-metrics-cluster/yaml/vmcluster.yaml
+  kubectl delete -n default -f https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/victoria-metrics-cluster/yaml/vmagent.yaml
+fi
```

Comment:
I don't get errors from this, but it does hang indefinitely and the pods don't get deleted.